en/releases/daily_build_new_features.md

```diff
@@ -14,9 +14,10 @@ Allows you to create mission items which specify altitude as height above terrai
 
 ### Corridor Scan
 
-![](../../assets/Plan/CorridorScan.jpg)
+![Corridor Scan](../../assets/Plan/corridor_scan.jpg) 
```

Comment:
This is daily build new features, but from what I can see the feature made it into QGC3.4. Should this be removed from here?

Reply:
Yup. Shouldn't be there

Reply:
Thanks. Removed in https://github.com/mavlink/qgc-user-guide/commit/6130af160578dd67a8a4e784e367abb641d5cdd2